api/stages_api.py

```diff
@@ -69,31 +71,6 @@ class StagesAPI(basehandlers.APIHandler):
       'rollout_platforms',
       'enterprise_policies']
 
-  def _stage_to_json_dict(self, stage: Stage) -> dict:
```

Comment:
Thanks for doing this.  I recently noticed that there was redundant code in these functions.